.gitlab-ci.yml

```diff
@@ -244,6 +244,7 @@ Integration:
         RUNNER:
           - aws/rhel-8-x86_64
           - openstack/rhel-8.5-x86_64
+          - aws/rhel-9.0-beta-nightly-x86_64
```

Comment:
I would rather use `openstack` for this, `aws` doesn't support nested virtualization and `libvirt` in RHEL doesn't officially support emulation so it may break at any time and no one will fix it (this happened in 8.5).